File: python/20260317/kutya.py
from allat import Allat


class Kutya(Allat):
    def __init__(self, nev: str, eletkor: int, bekerulesi_ev: int, orokbefogadhato: bool, fajta: str) -> None:
        super().__init__(nev, eletkor, bekerulesi_ev, orokbefogadhato)
        self.fajta = fajta

    def hangot_ad(self) -> str:
        return "Vau vau!"

    def adatok_szovegesen(self) -> str:
        return f"Kutya - {super().adatok_szovegesen()}, fajta: {self.fajta}, hang: {self.hangot_ad()}"

File: C#/20260317/Allatmenhely/Kutya.cs
namespace Allatmenhely;

public class Kutya : Allat
{
    public string Fajta { get; }

    public Kutya(string nev, int eletkor, int bekerulesiEv, bool orokbefogadhato, string fajta)
        : base(nev, eletkor, bekerulesiEv, orokbefogadhato)
    {
        Fajta = fajta;
    }

    public override string HangotAd()
    {
        return "Vau vau!";
    }

    public override string AdatokSzovegesen()
    {
        return $"Kutya - {base.AdatokSzovegesen()}, fajta: {Fajta}, hang: {HangotAd()}";
    }
}
